webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
```

Comment:
Does one of the updated packages require us to have a webpack config now?

Reply:
No I removed it in the latest commit. It was left over from trying to update react-scripts to 5.0.0